go/dbproxy/local_table.go

```diff
@@ -65,6 +65,44 @@ func localTableGetOne(c *gin.Context) {
 	c.JSON(http.StatusOK, gin.H{"result": "ok", "data": row})
 }
 
+func contains(array []string, el string) bool {
+	for _, x := range array {
+		if x == el {
+			return true
+		}
+	}
+	return false
+}
+
+func localTableGetByField(c *gin.Context) {
+	localTable := sql.GetLocalTable()
+	m, ok := sql.NewRow(c.Param("name"))
+	if !ok {
+		c.JSON(http.StatusNotFound, gin.H{"error": "table name not found"})
+		return
+	}
+
+	userID, err := parseUserID(c)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+	}
+
+	field := c.Param("field")
+	if !contains(m.Fields(), field) {
+		c.JSON(http.StatusNotFound, gin.H{"error": "invalid field"})
+		return
+	}
+
+	value := c.Param("value")
```

Comment:
Can `value` be empty or we should check `len(value) > 0`?

Reply:
Would the URL pattern match at all if the value is empty?

Reply:
The URL pattern will not match since value the last component. 
So the code looks good.